ccloud/docs/beginner-cloud.rst

```diff
@@ -0,0 +1,131 @@
+
+.. _beginner-cloud:
+
+Confluent Cloud CLI
+===================
+
+Overview
+--------
+
+You can use `Confluent Cloud CLI
+<https://docs.confluent.io/current/cloud/cli/install.html#ccloud-install-cli?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+to interact with your `Confluent Cloud
+<https://confluent.cloud/?utm_source=github&utm_medium=demo&utm_campaign=ch.examples_type.community_content.beginner-cloud>`__
+cluster.
+
```

Comment:
Need a bit more of an introduction, e.g. this tutorial will show you how ....

Reply:
Updated